alica_dummy_proxy/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 3.10)
 project(alica_dummy_proxy)
 
-add_compile_options(-std=c++17)
+add_compile_options(-std=c++17 -Werror)
```

Comment:
Should probably add in the cmake config file that is shared by all alica package soon to be moved by one of Dhananjay's PRs